messages/store.json

```diff
@@ -118,6 +118,34 @@
             "done": "Done"
         }
     },
+    "extension": {
+        "register": {
+            "cmdDescription": "Register an extension with one command",
+            "apexClassNameFlagDescription": "Name of the Apex class that needs to be extended",
+            "errApexClass": "Apex class doesn't exist in webstore.",
+            "nameAlreadyExists": "Registered-Extension name already exists",
```

Comment:
super nit: should we use either `Registered Extension name` (user friendly format) or `registered-extension-name` (to match parameter format) in the error message. Looking at other commands they seem to use the user friendly format. 

e.g.
```
sfdx commerce:store:display -n ahc
ERROR running commerce:store:display:  No store matched ahc %s %s %s

Reply:
nit: should we add a colon at the end of the message?
Current:
`ERROR running commerce:extension:register:  Registered-Extension name already exists EXTENSION_NAME`
Expected
`ERROR running commerce:extension:register:  Registered-Extension name already exists: EXTENSION_NAME`